egs/librispeech/ASR/zipformer/train.py

```diff
@@ -1217,7 +1216,8 @@ def remove_short_and_long_utt(c: Cut):
 
         return True
 
-    train_cuts = train_cuts.filter(remove_short_and_long_utt)
+    if not params.stateless_sampler:
```

Comment:
this look problematic if you want to compare apples to apples, I can add a `sampler.filter(predicate)` function to the stateless sampler to allow the filtering to happen if you'd like